04_discussion.Rmd

```diff
@@ -10,7 +10,7 @@ While the opportunities to champion reproducible workflows are endless, we remai
+DFO collects a wealth of data, yet in various instances, datasets not initially intended for spatial analysis contain useful information for our Report to strengthen decision-making. Our work focuses on automation, and these datasets require significant data wrangling to identify, extract, and integrate this information into a usable form [@Furche2016; @Kandel2011]. For example, funding agencies often focus on threats facing Species at Risk as part of their research priorities. There is great potential for this tool to identify and provide this threat information alongside the Species at Risk observed within a search area, and we have started this work within our Human Activities module. However, many of the documents mentioned in the SARA recovery planning process contain threat information with various levels of detail. Threat information can be mentioned in paragraphs, tables, sentences, or point form, and range from highly qualitative to quantitative, and these documents have gone through multiple generations of changes to further refine and define how to characterize threats [@DFO2007; @DFO2010; @DFO2014]. We are currently in discussion with our collaborators for ways to best summarize threat information so it is comprehensive, current, and accurate, to achieve these goals. We have also encountered additional challenges with non-standardized formats when undergoing an additional data mining and wrangling exercise to extract Species at Risk data from Section 73 permits which are used when a human activity affects listed wildlife species (e.g., developments, research purposes; SARA s.73, 2002). They often contain a mix of digital and hand-written documentation, have coordinate information in multiple formats, or in several instances, provide limited to no spatial context. Other sectors have recognized these challenges and are in discussion to standardize these approaches in future years. Advancements in big data and reproducible approaches cause excitement, yet there are stark warnings that irresponsible data wrangling within reproducible workflows can be misleading or draw incorrect conclusions [@Leek2015]. Aware of these challenges, we continue to be careful, measured, and consistently consult species experts to guide our work, and provide credible information without losing important context.
```

Comment:
The change I made was swapping: However, threats are mentioned in many of the documents mentioned in the SARA recovery planning process, and contain various levels of detail.  ===> However, many of the documents mentioned in the SARA recovery planning process contain threat information with various levels of detail.
Not sure why github is not highlighting that though.